python/mxnet/autograd.py

```diff
@@ -112,6 +137,34 @@ def pause(is_train=False):
     return RecordingStateScope(False, is_train)
 
 
+def override_train():
```

Comment:
set_training is enough for now. We can add train()/predict() after getting more user feedback